apps/flipper_auth/pubspec.yaml

```
@@ -30,6 +30,7 @@ dependencies:
   flipper_models: any
   flipper_services: any
   sqlite3: ^3.1.0
+  sqlite3: ^3.1.0
```

YAMLlint (1.37.1)

[error] duplication of key "sqlite3" in mapping

(key-duplicates)